Basic-Car-Maintenance-UITests/BasicCarMaintenanceUITests.swift

```diff
@@ -18,6 +18,8 @@ final class BasicCarMaintenanceUITests: XCTestCase {
     }
     
     func testAddVehicle() {
```

Comment:
This doesn't pass for me, I get this error.
I erased the simulator, and deleted all the data from the emulator
```
Running tests...
Failed to send CA Event for app launch measurements for ca_event_type: 0 event_name: com.apple.app_launch_measurement.FirstFramePresentationMetric
Failed to send CA Event for app launch measurements for ca_event_type: 1 event_name: com.apple.app_launch_measurement.ExtendedLaunchMetrics
    t =      nans Interface orientation changed to Portrait
Test Suite 'BasicCarMaintenanceUITests' started at 2024-10-27 16:26:30.306.
Test Case '-[Basic_Car_Maintenance_UITests.BasicCarMaintenanceUITests testAddVehicle]' started.
    t =     0.00s Start Test at 2024-10-27 16:26:30.307
    t =     0.13s Set Up
    t =     0.13s     Open com.icyappstudio.Basic-Car-Maintenance
    t =     0.13s         Launch com.icyappstudio.Basic-Car-Maintenanc
warning: (arm64) /Users/mikaelacaron/Library/Developer/Xcode/DerivedData/Basic-Car-Maintenance-hasiwwufjtgocfeyrzxbvwqlinjv/Build/Products/Debug-iphonesimulator/Basic-Car-Maintenance.app/Basic-Car-Maintenance empty dSYM file detected, dSYM was created with an executable with no debug info.
e
    t =     1.58s             Setting up automation session
    t =     2.48s             Wait for com.icyappstudio.Basic-Car-Maintenance to idle
    t =     3.97s Tap "dismiss popup" Other
    t =     3.97s     Wait for com.icyappstudio.Basic-Car-Maintenance to idle
    t =     4.35s     Find the "dismiss popup" Other
    t =     5.47s         Find the "dismiss popup" Other (retry 1)
    t =     6.55s         Find the "dismiss popup" Other (retry 2)
    t =     6.61s Requesting snapshot of accessibility hierarchy for app with pid 7176
/Users/mikaelacaron/Documents/Apps/Basic-Car-Maintenance/Basic-Car-Maintenance-UITests/Extensions/XCUIApplication-dismissPopup.swift:13: error: -[Basic_Car_Maintenance_UITests.BasicCarMaintenanceUITests testAddVehicle] : Failed to tap "dismiss popup" Other: No matches found for Elements matching predicate '"dismiss popup" IN identifiers' from input {(
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other, label: 'Dismiss',
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other, label: 'Filter',
    Other, identifier: 'plus', label: 'Add',
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other,
    Other, label: 'Vertical scroll bar, 1 page', value: 0%,
    Other
)}
Possibly caused by runtime issues:
Automation type mismatch: computed Other from legacy attributes vs ActivityIndicator from modern attribute. Input attributes and values: {
    "XC_kAXXCAttributeAutomationType" = 36;
    "XC_kAXXCAttributeElementBaseType" = UIResponder;
    "XC_kAXXCAttributeElementType" = "SwiftUI.AccessibilityNode";
    "XC_kAXXCAttributeTraits" = 8589935104;
}
Automation type mismatch: computed Other from legacy attributes vs StaticText from modern attribute. Input attributes and values: {
    "XC_kAXXCAttributeAutomationType" = 48;
    "XC_kAXXCAttributeElementBaseType" = UILabel;
    "XC_kAXXCAttributeElementType" = UILabel;
    "XC_kAXXCAttributeTraits" = 8590000128;
}
See test report attachments for more detail.
    t =     6.83s Tear Down
Test Case '-[Basic_Car_Maintenance_UITests.BasicCarMaintenanceUITests testAddVehicle]' failed (7.091 seconds).
Test Suite 'BasicCarMaintenanceUITests' failed at 2024-10-27 16:26:37.398.
	 Executed 1 test, with 1 failure (0 unexpected) in 7.091 (7.092) seconds
```